Purchases/Attribution/RCAttributionFetcher.h

```diff
@@ -29,7 +29,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (nullable NSString *)identifierForVendor;
 
-- (void)adClientAttributionDetailsWithCompletionBlock:(RCAttributionDetailsBlock)completionHandler;
+- (void)afficheClientAttributionDetailsWithCompletionBlock:(RCAttributionDetailsBlock)completionHandler;
```

Comment:
why affiche? affiche = attach in french? wondering how we can document this while avoiding the-words-that-must-not-be-named.

or maybe we could do a funny misspelling like  `AhdClyent`?

Reply:
+1. That'd make it easier for devs to track down the original class. `adKlient`?

Reply:
Lolll good idea
`adKlient` 
I kinda have insight into how Apple does scanning for words (I was part of the China Online Store launch). At least there, they look for permutations of the same word using different letters that have the same sound or are similar. So `AdKlient` would get caught, `@dClient` `@dKlient` as well. The word needs to be different enough from the word they are scanning for. 

This is all just a guess, of course.

Reply:
So actually, I'm pretty sure `AddClyent` would also get caught.